### PR TITLE
[cli] add rooch move new --address var.

### LINKED ISSUE #271: [cli] `rooch move new` command allow developer to pass package address
The `rooch move new` generates a random address for a new project. We must allow the developer to pass the package address or use the active one.

### PR DESCRIPTION
**Pull request by feliciss**
Resolve #271

### CODE REVIEW

File: crates/rooch/src/commands/move_cli/commands/new.rs [hunk 1]
```diff
@@ -21,13 +21,25 @@ const ROOCH_FRAMEWORK_PKG_PATH: &str = "{ git = \"https://github.com/rooch-netwo
...
 
 impl New {
     pub fn execute(self, path: Option<PathBuf>) -> anyhow::Result<()> {
         let name = &self.new.name.to_lowercase();
+        let address = if let Some(account_address) = &self.account_address {
+            // Existing account address is available
+            account_address.to_hex_literal()
+        } else {
+            // Existing account address is not available, generate a new random address
+            AccountAddress::random().to_hex_literal()
```
Comment (jolestar) [inline crates/rooch/src/commands/move_cli/commands/new.rs:40]: if the account address is None, use the default account.

Comment (feliciss) [reply crates/rooch/src/commands/move_cli/commands/new.rs]: The account is now the default account from config.

**Review by pause125 — APPROVED** [review]

File: crates/rooch/src/commands/move_cli/commands/new.rs [hunk 2]
```diff
@@ -21,13 +25,53 @@ const ROOCH_FRAMEWORK_PKG_PATH: &str = "{ git = \"https://github.com/rooch-netwo
...
+    #[clap(long = "address", short = 'a')]
+    account_address: Option<AccountAddress>,
+
     #[clap(flatten)]
     pub new: new::New,
 }
 
 impl New {
-    pub fn execute(self, path: Option<PathBuf>) -> anyhow::Result<()> {
+    async fn get_active_account_address_from_config(&self) -> Result<String, RoochError> {
```
Comment (jolestar) [inline crates/rooch/src/commands/move_cli/commands/new.rs:37]: You should use `WalletContextOptions` to build the WalletContext, Ref

https://github.com/rooch-network/rooch/blob/17e30cbfe5bc4de3a16ba3440b2d874b1a899588/crates/rooch/src/commands/move_cli/commands/build.rs#L18-L35

Comment (feliciss) [reply crates/rooch/src/commands/move_cli/commands/new.rs:41]: Added WalletContextOptions.

**Review by jolestar — APPROVED** [review]

**Comment by feliciss** [comment]
Formatted the code and resubmit https://github.com/rooch-network/rooch/commit/536a62d7b1bd4fda7eb543b62b668fc828e22aac

**Review by jolestar — APPROVED** [review]

**Pull request merged**